db/deprecated/engine.py

```diff
@@ -21,13 +21,18 @@ def create_future_engine_with_custom_types(
 def create_future_engine(
         username, password, hostname, database, port, *args, **kwargs
 ):
+    query = {}
+    if hostname.startswith("/") or hostname.startswith("%2F") :
```

Comment:
```suggestion
    if hostname.startswith("/") :
```

I don't think we're supporting encoded strings here anymore, but @pavish would know better.